packages/p2p/src/components/my-ads/create-ad-form-payment-methods.jsx

```diff
@@ -42,6 +38,7 @@ const CreateAdFormPaymentMethods = ({ is_sell_advert, onSelectPaymentMethods })
         }
     };
 
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Why is this added here?

Reply:
Eslint was commenting on it previously being unused while its being used as a prop in the new component (edit: just ran `npm test` and now its passed, will remove the comment asap)